telemetry/grafana/dashboards/openfga.json

```diff
@@ -103,14 +103,7 @@
             "__systemRef": "hideSeriesFrom",
```

Comment:
Instead of hiding the `CreateStore` series from the panel, just omit the overrides altogether so that all timeseries show by default.

`overrides: []`

This is what is being done in the p99 request latency panel.

Good catch though! That's a bit of an annoyance.

Reply:
Ah i thought we wanted to keep this filter (for whatever reason). Removed!